crates/subspace-bundle-checker/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "subspace-bundle-checker"
```

Comment:
Although the check is performed at txn pool, within this crate there is nothing related to txn pool, `subspace-tx-pool-bundle-validator` could be confusing, how about `subspace-bundle-validator`?

Reply:
Although the check is performed at txn pool, within this crate there is nothing related to txn pool, `subspace-tx-pool-bundle-validator` could be confusing, how about `subspace-bundle-validator`?

Reply:
`subspace-bundle-validator` is not concrete/accurate as the bundle validation takes place in both the tx pool and runtime, `subspace-bundle-validator` makes me feel that all the bundle validations are covered in this place, but it's not in fact. Why `subspace-tx-pool-bundle-validator` is confusing? It can be even part of `subspace-transaction-pool` as a module.

Reply:
> Why subspace-tx-pool-bundle-validator is confusing?

As a standalone crate, its name should reflect its functionality instead of where it is used outside.

> It can be even part of subspace-transaction-pool as a module.

Well, this is a better option compared to `subspace-tx-pool-bundle-validator` I think.

Reply:
Well, `subspace-tx-pool-bundle-validator` exactly tells the function it serves, which is the bundle validator inside the tx pool and this is everything this crate provides.

Reply:
Bundle validation is the function this crate provides, and where to use it whether in txn pool, block import, or any other place is a decision made out of this crate. If we want to emphasize that it should only be used in txn pool then making it a part of `subspace-transaction-pool` as a module is indeed a better option I think. But I will change it to `subspace-tx-pool-bundle-validator` if you insist.

Reply:
>it should ever only be used in txn pool

This is true, the bundle validation logic this crate is scoped to the tx pool only and not intended to be used in any other places (at least for now).

Either it's a seperate crate or a module of `subspace-transaction-pool` is fine to me, but if you make it a crate, I'd expect it's named in a non-confusing way.